db/seeds.rb

```diff
@@ -16,6 +16,14 @@ def self.create_charges_for_user(user)
   Charge.create(operation_date: Date.today, amount: 10, description: "na nalog", user_id: user.id)
 end
 
+def self.plans_for_user(user)
+  Plan.create(start_date: Date.today, end_date: Date.today, amount: 10, user_id: user.id, category: food)
+end
+
+def self.create_plan_charges_for_user(user)
+  Plan_charge.create(operation_date: Date.today, amount: 1, description: "moar money", user_id: user.id)
```

Comment:
Wrong syntax! You haven't `Plan_charge` but  `PlanCharge` model. Also PlanCharge should be linked with plan through `plan_id`